schema-registry-mock/src/main/java/com/bakdata/schemaregistrymock/SchemaRegistryMock.java

```diff
@@ -96,17 +101,19 @@
     private static final String ALL_SUBJECT_PATTERN = "/subjects";
     private static final String SCHEMA_PATH_PATTERN = "/subjects/[^/]+/versions";
     private static final String SCHEMA_BY_ID_PATTERN = "/schemas/ids/";
+    private static final String SCHEMA_VERSION_PATTERN = "/subjects/[^/]+\\?deleted=true";
```

Comment:
Why is `deleted=true` part of the pattern?

Reply:
The deleted parameter is set to `true` when calling `getVersion(subject, schema)` and to false when calling `getId(subject, schema)`.
Originally, I did not plan to include support for id requests and I wanted to make sure that there are no unexpected responses. Especially, since the parameter is not documented anywhere. 
I looked into the behavior of a real schema registry and it responds as expected: When set to false, querying a deleted schema (not subject!) returns `{"error_code": 40403, "message": "Schema not found"}. Otherwise, it returns the complete schema information.
I'll remove the parameter from the pattern and add the logic for both cases in the transformer.